src/Controls/src/Core/Compatibility/Handlers/NavigationPage/iOS/NavigationRenderer.cs

```diff
@@ -780,10 +792,34 @@ void RefreshBarBackground()
 				}
 				else
 				{
-					if (_currentBarBackgroundColor?.Alpha < 1f)
-						navigationBarAppearance.ConfigureWithTransparentBackground();
+					// If user explicitly set translucent property, respect that value
+					if (isNavigationBarTranslucentExplicitlySet)
```

Comment:
Duplicate logic in both iOS 13+ and pre-iOS 13 branches, could creare a method and reuse it?

Reply:
@jsuarezruiz, the way translucency/opacity is applied differs between iOS 13+ (UINavigationBarAppearance) and pre-iOS 13 (BarTintColor / SetBackgroundImage), so I don’t think we can fully unify those blocks without introducing another branch inside the shared method. I extracted the common decision logic into a method but kept the application code separate since the APIs differ and require a different approach.